proto/toit/api/organization.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package toit.api;
+
+import "toit/model/organization.proto";
+
+option go_package = "github.com/toitware/api.git/toit/api";
+option java_package = "io.toit.proto.toit.api";
+option java_outer_classname = "OrgProto";
+
+service OrganizationService {
+  rpc SetDefaultSDK(SetDefaultSDKRequest) returns (SetDefaultSDKResponse) {} 
+  rpc DefaultSDK(DefaultSDKRequest) returns (DefaultSDKResponse) {} 
+  rpc ListUsers(ListUsersRequest) returns (ListUsersResponse) {}
+  rpc CreateUser(CreateUserRequest) returns (CreateUserResponse) {}
+  rpc DeleteUser(DeleteUserRequest) returns (DeleteUserResponse) {}
```

Comment:
With these endpoints we should deprecate the ones found in `proto/toit/api/user.proto`